mods/ts/rules/nod-vehicles.yaml

```diff
@@ -547,3 +547,84 @@ STNK:
 	Explodes@ELITE:
 		RequiresCondition: rank-elite
 		Weapon: UnitExplode
+
+SGEN:
+	Inherits: ^Tank
+	Inherits@VOXELS: ^VoxelActor
+	Inherits@selection: ^SelectableSupportUnit
+	-AppearsOnRadar:
+	Buildable:
+		Queue: Vehicle
+		BuildPaletteOrder: 90
```

Comment:
it should be 100+. This is only buildable after tech centre so units with lower prerequisites such as Nod Artillery (100) should come first